react-app/src/components/property/PropertyDetail.tsx

```diff
@@ -82,6 +84,21 @@ const PropertyDetail = (props: IPropertyDetail) => {
       api.buildings.getBuildings({ pid: parcel?.parsedBody?.PID, includeRelations: true }),
   );
 
+  useEffect(() => {
+    const fetchLinkedProjects = async () => {
+      if (parcelId) {
+        const projects = await api.properties.getLinkedProjectsToProperty({ parcelId: parcelId });
+        setLinkedProjects(projects);
+      } else if (buildingId) {
+        const projects = await api.properties.getLinkedProjectsToProperty({
+          buildingId: buildingId,
+        });
+        setLinkedProjects(projects);
+      }
+    };
```

Comment:
If the parcelId or buildingId is undefined, is it possible to combine these statements?
Would suggest testing this locally before committing.
```suggestion
      const projects = await api.properties.getLinkedProjectsToProperty({ 
        parcelId,
        buildingId,
      });
      setLinkedProjects(projects);    
    };
```

Reply:
It is possible, but I had wrap api call in a try catch block, as without it won't return anything